sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -61,95 +171,199 @@ Table 1. Values of the `SYCL_EXT_ONEAPI_GRAPH` macro.
 |1     |Initial extension version. Base features are supported.
 |===
 
-== SYCL Graph Terminology
+=== SYCL Graph Terminology
+
+:explicit-memory-ops: https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#subsec:explicitmemory
 
 Table 2. Terminology.
 [%header,cols="1,3"]
 |===
-|Concept|Description
-|graph| Class that stores structured work units and their dependencies.
-|node| The unit of work. Can have different attributes.
-|edge| Dependency between work units. Happens-before relation.
+| Concept | Description
+
+| Graph
+| `command_graph` class that stores structured commands and their dependencies.
+
+A SYCL graph is a collection of commands (nodes) and their dependencies (edges).
+From the SYCL perspective, this graph will be acyclic and directed (DAG) as
+users cannot express a cycle in the core SYCL API.
+
+| Node
+| A command, which can have different attributes.
+
+When recording a queue to construct a graph, nodes in a SYCL graph represent
```

Comment:
I've split this up into a table with the generic description, followed by subsections for each of the modes with their own table for how nodes & edges are realized. Which I agree looks cleaner rather than mixing these all in the same description.

Reply:
One thing I've said is that in the explicit API subsection is that edges can be created from buffer accessors even if the user doesn't specify any node dependencies. Could you confirm if this is the case or not?

e.g
```cpp

 sycl::buffer buf;
 auto n1 = graph.add(
        [&](sycl::handler& cgh){
        auto acc = buf.get_access<access::mode::read_write>(cgh);
         cgh.parallel_for(sycl::range<1>{n},
		  [=](item<1> id){acc[id]=1; }
		 );
    });

auto n2 = graph.add(
        [&](sycl::handler& cgh){
	 auto acc = buf.get_access<access::mode::read_write>(cgh);
         cgh.parallel_for(sycl::range<1>{n},
		  [=](item<1> id){acc[id] *= 2; }
		 );
    });

// Does n2 depend on n1 because of the buffer accessor?

```

Reply:
Yes that should be supported.

Reply:
cool, makes sense but wanted to check 👍